includes/wikia/nirvana/WikiaApp.class.php

```diff
@@ -441,7 +441,7 @@ public function onArticleViewHeader( &$article, &$outputDone, &$useParserCache )
 		$title = $article->getTitle();
 
 		$namespace = $title->getNamespace();
-		if ( !empty( $this->namespaceRegistry[$namespace] ) &&  $title->exists() ) {
+		if ( !empty( $this->namespaceRegistry[$namespace] ) ) {
```

Comment:
Finishing cleanup of this line:
https://github.com/Wikia/app/commit/134d1e8d5be60279d542d9a2ef2942a3bd89c9f8#diff-4da689556b30d347058fa43546e6f09aL448